onnxruntime/python/tools/transformers/models/whisper/convert_to_onnx.py

```diff
@@ -169,6 +169,69 @@ def parse_arguments(argv=None):
     )
     parser.set_defaults(chain_model=True)
 
+    parser.add_argument(
```

Comment:
For the new arguments (and existing ones), it would be very helpful to create groups with `parser.add_argument_group` so it is clear which arguments need to be provided together. Can we do that for these new arguments?

Reply:
We could do it later.